imod/tests/test_mf6/test_mf6_wel.py

```diff
@@ -137,35 +140,120 @@ def test_to_mf6_pkg__high_lvl_transient(basic_dis, well_high_lvl_test_data_trans
     np.testing.assert_equal(mf6_ds["rate"].values, rate_expected)
 
 
-def test_clip_box__high_lvl_stationary(well_high_lvl_test_data_stationary):
+class ClipBoxCases:
```

Comment:
I like these cases! I think you should add some cases where ``top`` and ``bottom`` are provided as grids to see what I'm talking about in my other comment ;-)

FYI: In the ``parameterizable_basic_dis`` fixture, ``top`` and ``bottom`` are not created as grids:

```python
    top = xr.DataArray(z[:-1], coords={"layer": layers})
    bottom = xr.DataArray(z[1:], coords={"layer": layers})
```